pkg/webhook/mutating/pod.go

```diff
@@ -133,6 +115,8 @@ func PodCreateMutation(ctx context.Context, req *admission.Request, handler *Han
 			patchSelectorToPod(pod, map[string]string{
 				constants.LabelDualStackAddressQuota: constants.QuotaNonEmpty,
 			})
+		default:
```

Comment:
actually, ip family is validated in validating webhook before mutation, so `default` is unreachable.

Reply:
done